code/script.js

```diff
@@ -253,91 +257,136 @@ const setSecret = () => {
 const start = () => {
   // Here we're setting charactersInPlay array to be all the characters to start with
   charactersInPlay = CHARACTERS
-  // What else should happen when we start the game?
```

Comment:
I was told on my last review that we need to remove the comments from the final code. But I totally left them too for didactic purposes. 😁

Reply:
Yes, remember to always clean your code from all console.log()s and commented out code blocks and tips left for you by teachers 🗡️ ✂️